### DISCUSSION TITLE
Buttons in ink

### DISCUSSION THREAD

**Discussion by Rxann** [vadimdemedes/ink#528] [Q&A]
Are there a way to use a button component or element in ink?

**Comment by vadimdemedes** [comment]
Nope, there isn't anything like this. Usually CLIs don't have buttons.

**Reply by Rxann** [reply]
Ok, is there any way to make a sort of routing where it would render different components on an input?

**Reply by vadimdemedes** [reply]
You can use `if..else` statements if you need to display different components on condition.

**Reply by Rxann** [reply]
ok thank you
